taqueria-plugin-mock/index.js

```diff
@@ -1,21 +1,29 @@
 const {Plugin, Task, Option} = require('@taqueria/node-sdk')
+
 Plugin.create(i18n => ({
     schema: "1.0",
     version: "0.1",
+    alias: "mock",
     tasks: [
         Task.create({
             task: "ping",
-            command: "ping",
+            command: "ping [sourceFile]",
             aliases: ["p", "ping-test"],
             description: "Return predefined result for CLI integration testing",
             options: [
                 Option.create({
                     shortFlag: "e",
-                    flag: "return error ",
-                    description: "The "
-                }),
+                    flag: "error",
+                    description: "Mock plugin is used for integration testing"
```

Comment:
I think the description should be changed to "Forces the task to return an error.", as this description is for the option, not the plugin itself.